local/bin/py/synthetics_worker_version.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
it looks like this is doing something very similar to `version_getter.py` which is referenced in the github workflow.    is this one needed as well?

Reply:
Ah yeah sorry, I thought I'd deleted/renamed this!